app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+
+  has_secure_password
+
+  validates_presence_of :name
+  validates_presence_of :email
+  validates_uniqueness_of :email
+  validates_presence_of :password
+  # validates_confirmation_of :password
```

Comment:
Is this still needed for something later?

Reply:
# validates_confirmation_of :password 
this line you mean?

Reply:
Yeah because it currently is just commented out